charts/tidb-cluster/templates/monitor-deployment.yaml

```diff
@@ -94,6 +94,17 @@ spec:
       - name: grafana
         image: {{ .Values.monitor.grafana.image }}
         imagePullPolicy: {{ .Values.monitor.grafana.imagePullPolicy | default "IfNotPresent" }}
+        lifecycle:
+            postStart:
```

Comment:
I don't whether grafana can discover dashboard json dynamically at runtime, but there is a race condition of `postStart` hook: kubelet cannot ensure the execution order of `postStart` hook and container entrypoint.
So if grafana could discover dashboard jsons at runtime, this is safe. Otherwise we have to ensure the unzip logic executed before grafana runs by init-container or other mechanisms.

Reply:
Yes, it is. The Grafana can discover and load dashboard json files at runtime.

Reply:
When Grafana starts, it will update/insert all dashboards available in the configured path. Then later on poll that path every updateIntervalSeconds and look for updated json files and update/insert those into the database
More detail in grafana document